app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+  has_one :user
```

Comment:
We don't need `has_one :user` association. This is causing an error to be thrown while trying to create a comment